### ISSUE TITLE
Break out the engine into a separate repo

### ISSUE THREAD

**Issue by dlrobertson** [anidata/palantiri#5]
`src/core/engine.py` and the likes could be easily used for future projects and might save us time in the future if we were to have to create another crawler. We should break it out into its own python module. Depending on how much is split out into a separate issue could make this more or less difficult. I'm happy to mentor anyone who dares pick this issue up :smile:

**Comment by wdm0006** [comment]
Seems like this would partially depend on #4.  It may be that the line between general purpose and project specific is farther out than just engine.py.

Regardless, I'd be happy to take this on and get the repo set up and everything in the meantime.  It may also be a good opportunity to hammer out some of the other standards like contributing.md, licensing, etc.

**Comment by wdm0006** [comment]
Ok, a new repo has been created here: https://github.com/anidata/rasp, once it is ready to be released we can make a change here to import from there and deprecate the code currently in palantiri.

**Comment by dlrobertson** [comment]
:+1: Thanks. The move to using `rasp` in `palantiri` can/should be a separate issue.
